cookbook/override-dir-structure.rst

```diff
@@ -36,5 +36,23 @@ generated it with ``npm install`` and ``npm run build`` into your new directory.
                 sulu_admin:
                     json_manifest_path: "%kernel.project_dir%/public/your/new/path/manifest.json"
 
+
+Load Snippets from a Subfolder
```

Comment:
Thanks for your PR!

I would prefer creating a `SnippetBundle` documentation (similar to the `LocationBundle` documentation, see also https://github.com/sulu/sulu-docs/pull/613#discussion_r547400063) and add this section there. I think that is the better approach if we (you 😉 ) want to extend the documentation in the future 🙂